pkg/backtest/matching.go

```diff
@@ -47,6 +47,12 @@ func init() {
 	}
 }
 
+type Param struct {
+	callbacks []func(types.KLine, *ExchangeDataSource)
```

Comment:
move this callback to ExchangeDataSource?

Reply:
done